packages/router/src/grpc/view-protocol-server/note-by-commitment.ts

```diff
@@ -1,33 +1,34 @@
 import type { Impl } from '.';
 import { servicesCtx } from '../../ctx';
+import { watchSubscription } from './util/watch-subscription';
 
-import {
-  NoteByCommitmentResponse,
-  SpendableNoteRecord,
-} from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/view/v1alpha1/view_pb';
+import { SpendableNoteRecord } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/view/v1alpha1/view_pb';
 
 import { ConnectError, Code } from '@connectrpc/connect';
 
-export const noteByCommitment: Impl['noteByCommitment'] = async (req, ctx) => {
+export const noteByCommitment: Impl['noteByCommitment'] = async (
+  { noteCommitment: findNoteCommitment, awaitDetection },
```

Comment:
Same here as below — I feel like the earlier name is clearer. If you want to emphasize that we're looking for this note commitment, you could call it `noteCommitmentToFind`. But right now it sounds like a function name.